web3/inpage.js

```diff
@@ -174,14 +49,11 @@ if (!window.chrome || !window.chrome.extension) {
 
       // Else, add a full web3 instance
       if (!web3.injectedWeb3) {
-        const rawWeb3 = require('web3/dist/web3.min.js');
-        eval(rawWeb3); // eslint-disable-line no-eval
-
-        const injectedWeb3 = new window.Web3(web3.currentProvider);
-        web3.injectedWeb3 = injectedWeb3;
-        // Expose object directly
-        window.web3 = injectedWeb3;
-        return injectedWeb3[name];
+        require('./lib');
```

Comment:
Yes, but I think it doesn't do that much during execution, does it? I don't see that much of a gain when lazy-instantiating `web3` and can think of scenarios when our approach fails, like:

```
if ('eth' in web3) {
 // want trigger a proxy, but should return `true` for standard Web3.
}
```